README.md

```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01 
+# **Jonathan Jacob**
```

Comment:
FYI placing this item in a header will automatically bold it so the bolding isn't necessary here.